src/main/java/io/arlas/server/core/FluidSearch.java

```diff
@@ -102,6 +104,9 @@
     public static final String RANGE_ALIASES_CHARACTER = "$";
     public static final String TIMESTAMP_ALIAS = "timestamp";
 
+    public static final String MIN_VALUE_OF_FILED = "min_field_values";
```

Comment:
Should be `field_min_value`, shouldn't it?

Reply:
Yes, it's better